src/test/java/org/candlepin/controller/ContentAccessManagerDBTest.java

```diff
@@ -62,15 +59,22 @@ public class ContentAccessManagerDBTest extends DatabaseTestFixture {
     private static final String ENTITLEMENT_MODE = ContentAccessMode.ENTITLEMENT.toDatabaseValue();
     private static final String ORG_ENVIRONMENT_MODE = ContentAccessMode.ORG_ENVIRONMENT.toDatabaseValue();
 
-    @Inject private ContentAccessCertificateCurator caCertCurator;
-    @Inject private KeyPairDataCurator keyPairDataCurator;
+    private ContentAccessCertificateCurator caCertCurator;
+    private KeyPairDataCurator keyPairDataCurator;
 
     private PKIUtility pkiUtility;
     private ObjectMapper objMapper;
     private X509V3ExtensionUtil x509V3ExtensionUtil;
 
     private EventSink mockEventSink;
 
+    @BeforeEach
```

Comment:
Have you verified if this is triggering a double-invocation of it? It being an override of a method that's already flagged with `@BeforeEach` makes this redundant or dangerous.